src/components/Menu/ActionList.vue

```diff
@@ -27,6 +27,7 @@
 		v-bind="state"
 		:container="menuIDSelector"
 		:aria-label="actionEntry.label"
+		:aria-activedescendant="!!currentChild"
```

Comment:
This should not be a boolean value, but the ID of the currently active (focused) element.

Reply:
Meaning this should be set on the menu toggle with the value of the active action button in the popover

Reply:
Right, seems I mixed that up, will have a look

Reply:
Adjusted to have a unique id matching with the menu entries.